pkg/services/live/live.go

```diff
@@ -748,7 +748,7 @@ func (g *GrafanaLive) HandleHTTPPublish(ctx *models.ReqContext, cmd dtos.LivePub
 		return response.Error(code, text, nil)
 	}
 	if reply.Data != nil {
-		_, err = g.node.Publish(cmd.Channel, cmd.Data)
+		err = g.Publish(ctx.OrgId, cmd.Channel, cmd.Data)
```

Comment:
What about another fix required to make things work - I mean returning `Data` in PublishReply inside `BroadcastRunner.OnPublish`? Was it unnecessary at the end?

Reply:
:facepalm: